src/modsim2/data/loader.py

```diff
@@ -398,11 +403,12 @@ def compute_similarity(self, only_train: bool = False):
     @staticmethod
     def _get_subset_data(
         subset_module: CIFAR10DMSubset,
-    ) -> Tuple[torch.Tensor, torch.Tensor]:
+    ) -> Tuple[np.array, np.array]:
         # force __getitem__ in CIFAR10 class for train and val
         # just take image rather than label
         train = torch.stack([item[0] for item in subset_module.dataset_train])
         val = torch.stack([item[0] for item in subset_module.dataset_val])
+        train, val = np.array(train), np.array(val)
```

Comment:
Instead of stacking via torch and then passing into `np.array`, we can just use `np.stack` in the first place:

```suggestion
        train = np.stack([item[0] for item in subset_module.dataset_train])
        val = np.stack([item[0] for item in subset_module.dataset_val])
```

Broadly, I think fine to have an output of either `torch.tensor` or `np.array` here, but from a clarity perspective it would be good to avoid changing it further in `compute_similarity`. Instead, we should have a default type that's clear in this fn, and in places where we need it (e.g. MMD needs `np.array` if that's not the default, OTDD needs `torch.tensor` if that's not the default) change it there.